vizro-core/tests/unit/vizro/actions/test_collapse_expand_container_action.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+from vizro.actions import collapse_expand_containers
+from vizro.managers import model_manager
+
+
+@pytest.mark.usefixtures("managers_one_page_two_containers")
+class TestCollapseExpandContainersActionInstantiation:
+    def test_invalid_duplicate_collapse_expand(self):
+        with pytest.raises(ValueError, match="Collapse and expand lists cannot contain the same elements!"):
```

Comment:
This would mention the explicit ID that's duplicated with the suggestion I made.